docs/options.md

```diff
@@ -247,14 +247,15 @@ environment variables will completely override any TOML configuration.
 
 > Override the auto-detected target platform
 
-Options: `auto` `linux` `macos` `windows`
+Options: `auto` `linux` `macos` `windows` `pyodide`
 
 Default: `auto`
 
 `auto` will build wheels for the current platform.
 
 - For `linux`, you need [Docker or Podman](#container-engine) running, on Linux, macOS, or Windows.
 - For `macos` and `windows`, you need to be running on the respective system, with a working compiler toolchain installed - Xcode Command Line tools for macOS, and MSVC for Windows.
+- For `pyodide` you need to be on an x86-64 linux runner and run cibuildwheel from Python 3.12.
```

Comment:
```suggestion
- For `pyodide` you need to be on an x86-64 linux runner and `python3.12` must be available in `PATH`.
```

I did test on macOS arm64 using cibuildwheel running python 3.11.
I'm not sure we want to encourage using anything else than linux x86_64 for CI but it might be worth mentioning that local builds can be done on other platforms if some are really supported.